schema/manifest_backwards_compatibility_test.go

```diff
@@ -137,12 +137,12 @@ func TestBackwardsCompatibilityManifest(t *testing.T) {
 		// "Accept: application/vnd.docker.distribution.manifest.v2+json" \
 		// https://registry-1.docker.io/v2/library/docker/manifests/sha256:888206c77cd2811ec47e752ba291e5b7734e3ef137dfd222daadaca39a9f17bc
 		{
-			digest: "sha256:888206c77cd2811ec47e752ba291e5b7734e3ef137dfd222daadaca39a9f17bc",
+			digest: "sha256:d901d5244c94db8151415bf3a1d3a0b10e7468e4997786b01c287cbcbaeca446",
```

Comment:
I don't think we want to bump this, but if we do we should probably bump the comment above or explain why the hashes are now different.